controllers/front/successHosted.php

```diff
@@ -76,7 +76,8 @@ public function postProcess()
             if (
                 !$authResponseBody->getLiability()->getLiabilityShift() &&
                 in_array($order->payment, SaferPayConfig::SUPPORTED_3DS_PAYMENT_METHODS) &&
-                $paymentBehaviourWithout3DS === SaferPayConfig::PAYMENT_BEHAVIOR_WITHOUT_3D_CANCEL
+                $paymentBehaviourWithout3DS === SaferPayConfig::PAYMENT_BEHAVIOR_WITHOUT_3D_CANCEL ||
```

Comment:
Add brackets to extract which condition should meet or logic, it is hard to read in current way.

Reply:
added one more set of brackets, idk there's a lot of conditions so it's hard to make it extra readable or maybe I just haven't seen a good example.